cypress/features/regression/dateInput.feature

```diff
@@ -31,15 +31,13 @@ Feature: Date Input component
   @positive
   Scenario: Change Date Input component minDate
     Given I open default "Date Input Test" component with "dateInput" json from "commonComponents" using "minDate" object name
-      And I set dateInput to today
-    When I choose date yesterday via DayPicker
+    When I set dateInput to minimum date
```

Comment:
I don't think it is a good idea, because we need to be dynamic

Reply:
Dynamic is why these tests are flakey in the first place, if you've got a better idea, I'm all ears 👍🏻